chart/values.yaml

```diff
@@ -47,13 +47,23 @@ processor:
     requests:
       cpu: "0.5"
       memory: "500Mi"
-  # See KAPELConfig.py for full details on required configuration.
-  config: |
-    SITE_NAME: "EXAMPLE-T2"
-    SUBMIT_HOST: "k8s.example.org:6443/namespace"
-    #BENCHMARK_VALUE: "15.0"
-    #NAMESPACE: "harvester"
-    #VO_NAME: "atlas"
+  # Config values for the processor initContainer. See KAPELConfig.py for full details on required configuration.
+  config:
+    # Name of the site being reported on
+    # SITE_NAME: "EXAMPLE-T2"
+    
+    # Uniquely identifying name for the cluster
+    # SUBMIT_HOST: "k8s.example.org:6443/namespace"
+    
+    # Namespace in the cluster where pilot jobs run
+    # NAMESPACE: "example-namespace"
+    
+    # Virtual Organization Name
+    VO_NAME: "example-vo"
+    # Works for a default installation of the bitnami/kube-prometheus Helm chart, otherwise use http://<service>.<namespace>:<port>
+    PROMETHEUS_SERVER: "http://kube-prometheus-prometheus.kube-prometheus:9090"
+    # Value to use for normalizing by CPU performance. Required for ssmsend output mode only
+    BENCHMARK_VALUE: "15.0"
```

Comment:
We can comment this too since 15.0 is just an arbitrary example; the real value should be set by the user.